client/src/main/kotlin/team/backend/dto/ChatDto.kt

```diff
@@ -0,0 +1,57 @@
+package team.backend.dto
+
+import com.fasterxml.jackson.annotation.JsonInclude
+
+class ChatDto {
+
+    companion object {
+        private const val chatModel = "gpt-3.5-turbo"
+        private const val chatRole = "user"
```

Comment:
I think it would be better to manage chatModel and chatRole with separate constants such as enum.

Reply:
As you said, I think it would be better to manage it with ENUM

Reply:
@BAEKJungHo 
I think there is a possibility that the model name will be changed to a value obtained from outside. so it is good to manage DB later